lib/moonchild.js

```diff
@@ -157,6 +162,17 @@ function applySafely(func, args) {
   }
 }
 
+function runDisplayHooks(tree) {
+  // Run the display hooks.
+  // TODO: This should be moved into a function that can be invoked by the
+  // editor plugin.
+  var hookArgs = getHookArgs(tree);
+  invokeHook('display', hookArgs);
+
+  // Run the render hooks.
+  invokeHook('render', hookArgs);
```

Comment:
Yeah once again, not sure what this was. Likely a failed attempt at refactoring.